d3ploy/demand_driven_deployment_inst.py

```diff
@@ -317,43 +353,63 @@ def target(incommod):
             else:
                 return self.commodity_supply[incommod]
 
-        if self.calc_method in ['arma', 'ma', 'arch']:
-            supply = CALC_METHODS[self.calc_method](target(commod),
-                                                    steps=self.steps,
-                                                    std_dev=self.supply_std_dev,
-                                                    back_steps=self.back_steps)
-        elif self.calc_method in ['poly', 'exp_smoothing', 'holt_winters', 'fft']:
-            supply = CALC_METHODS[self.calc_method](target(commod),
-                                                    back_steps=self.back_steps,
-                                                    degree=self.degree)
-        elif self.calc_method in ['sw_seasonal']:
+        if self.calc_method in ["arma", "ma", "arch"]:
+            supply = CALC_METHODS[self.calc_method](
+                target(commod),
+                steps=self.steps,
+                std_dev=self.supply_std_dev,
+                back_steps=self.back_steps,
+            )
+        elif self.calc_method in [
+            "poly",
+            "exp_smoothing",
+            "holt_winters",
+            "fft",
```

Comment:
what is with these commas in the last item of everything?

Reply:
The closing parenthesis should be here in place of the comma (and everywhere else there is a comma at the end of the set of keyword arguments.)

This is not acceptable and will not run (which would be clear if it were tested):

```
somefunction("variable", 
                        "other variable",
                        "last variable",
                        )
```

This is correct:


```
somefunction("variable", 
                        "other variable",
                        "last variable")
```

Reply:
I reverted all pep8 changes